metadata-db-app/src/migrations/051_add_created_at_to_granule_tables.clj

```diff
@@ -0,0 +1,51 @@
+(ns migrations.051-add-created-at-to-granule-tables
+  "Adds created_at column to granule tables."
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.mdb-migrate-helper :as h]
+   [config.migrate-config :as config]))
+
+(defn- add-created-at
+  []
+  (doseq [t (h/get-granule-tablenames)]
+    (h/sql (format "alter table %s add created_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL" t))
+    (h/sql (format "CREATE INDEX %s_c_i ON %s(created_at)" t t))))
```

Comment:
Good call - I used the monkey-see-monkey-do method for this migration (using this same migration for collections) and forgot to get rid of this. Fixed.